GloVe/download_glove_840b_300d.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+echo "Downloading GloVe Common Crawl (840B tokens, 2.2M vocab, cased, 300d vectors, 2.03 GB download)"
+wget http://nlp.stanford.edu/data/glove.840B.300d.zip
```

Comment:
Do a check here to see if the file already exists and the size matches. This is because it's a huge download so it's worth it to make the check. For smaller files, it's not a big issue.